### DISCUSSION TITLE
Windows Package Manager support

### DISCUSSION THREAD

**Discussion by qarkai** [ihhub/fheroes2#9925] [General]
Consider adding manifest for Windows installers to [winget repository](https://github.com/microsoft/winget-pkgs).

**Comment by oleg-derevenetz** [comment]
@qarkai I think you are free to do this, if you want to become a maintainer of such package and update it after each release.

**Comment by qarkai** [comment]
Hi @ihhub @oleg-derevenetz I'm going to add manifests
[fheroes2ResurrectionTeam.fheroes2.installer.yaml.txt](https://github.com/ihhub/fheroes2/files/6717553/fheroes2ResurrectionTeam.fheroes2.installer.yaml.txt)
[fheroes2ResurrectionTeam.fheroes2.locale.en-US.yaml.txt](https://github.com/ihhub/fheroes2/files/6717554/fheroes2ResurrectionTeam.fheroes2.locale.en-US.yaml.txt)
[fheroes2ResurrectionTeam.fheroes2.yaml.txt](https://github.com/ihhub/fheroes2/files/6717555/fheroes2ResurrectionTeam.fheroes2.yaml.txt)
Do you have have any objections/suggestions?

**Comment by oleg-derevenetz** [comment]
Hi @qarkai I never used winget, hence the question: are installers without an EV certificate suitable for it?

**Comment by qarkai** [comment]
Installer is filtered by SmartScreen but installation could be resumed.
Upd. New single manifest: [fheroes2ResurrectionTeam.fheroes2.yaml.txt](https://github.com/ihhub/fheroes2/files/6786054/fheroes2ResurrectionTeam.fheroes2.yaml.txt) (GitHub doesn't support yaml attachments).

**Comment by ihhub** [comment]
Moving this issue out of the scope as the team has no resources to maintain another distribution of the engine at the moment.

**Comment by ihhub** [comment]
Moving this issue into the Discussion section as this is not a part of the project.
